AMW_db_scripts/pom.xml

```diff
@@ -51,7 +51,13 @@
 			<artifactId>liquibase-slf4j</artifactId>
 			<version>${version.liquibase-slf4j}</version>
 		</dependency>
-		
+
+		<dependency>
+			<groupId>com.h2database</groupId>

```

Comment:
The driver is in the h2 profile and this should not be necessary. With `mvn compile -Pliquibase,h2 -Ddb=h2.test` it should work.

Reply:
You're right. I removed the dependency and altered the db update script